### ISSUE TITLE
IntersectIt does not work in QGIS v3

### ISSUE THREAD

**Issue by freeconst** [3nids/intersectit#14]
Hello. Are you planning to release a plug-in for version 3? Maybe there are some other tools for solving similar problems? In particular, I liked the intersection of circles. I tried to build a model for this function, but nothing happened. Unfortunately, my programming is bad.

**Comment by 3nids** [comment]
Hi,
There is no plan to upgrade this.
If you need this or a feature of the plugin in QGIS 3 and can support it, I would be happy to do so.

**Comment by ponceta** [comment]
@freeconst QGIS 3.X core should have almost all beloved features from intersectit in its advanced digitizing panel. If you find any missing please let us know!

**Comment by freeconst** [comment]
@ponceta Advanced digitizing panel not a bad tool (although I liked CadTools plugin in 2.x more), but there is no intersection of circles. I have to go out of the digitizing, build 2 circles with a given radius, and only then add a point on their intersection. Until I found another solution, it takes a lot of time.

**Comment by ppahv** [comment]
@freeconst Agree, but Advanced Digitizing Panel can't perform circle intersections. It'd be so nice to have an update for this plugin. Actually, this plugin and CadTools are the two tools that are missing from QGIS 3.0, and which prevent me from upgrading.

**Comment by ponceta** [comment]
@freeconst @ppahv Nice to here from you!

There's actually a bigger general problem with circle and curves. There is actually (as far as I know) no support for 2D curves intersection in QGIS : 

![snapping_curves](https://user-images.githubusercontent.com/8614223/53879963-0cce1e80-4010-11e9-80ea-2d7dc7f1a720.gif)

**Comment by freeconst** [comment]
@ponceta Yes, that is right. I turn on the centimeter grid and set a point on it. And this also takes time. Also, the curve is stored only in RAM, if you save it to disk, it becomes a polyline.

**Comment by ponceta** [comment]
@freeconst we financed a fix for intersections :

https://github.com/qgis/QGIS/pull/9379

This is available on QGIS 3.4.7 and will probably be a game changer on 3D and curve elements on QGIS.
